collection-scripts/gather_network_logs

```diff
@@ -17,7 +18,27 @@ function gather_multus_data {
 CLUSTER_NODES="${@:-$(oc get node -l node-role.kubernetes.io/master -oname)}"
 /usr/bin/gather_multus_logs $CLUSTER_NODES
 
-function gather_ovn_kubernetes_data {
+function gather_ovn_kubernetes_data_interconnect_mode {
+  echo "INFO: Gathering ovn-kubernetes DBs"
+  OVNKUBE_CONTROLLER_PODS=($(oc -n openshift-ovn-kubernetes get pods -l app=ovnkube-node -o=jsonpath='{.items[*].metadata.name}'))
+  # collect dbs from each node
+  for DB in "n" "s"; do
+    if [ "$DB" = "n" ]; then
+      DB_NAME="OVN_Northbound"
+      CONTAINER="nbdb"
+    else
+      DB_NAME="OVN_Southbound"
+      CONTAINER="sbdb"
+    fi
+    for OVNKUBE_CONTROLLER_POD in "${OVNKUBE_CONTROLLER_PODS[@]}"; do
+      echo "INFO: Gathering "${DB_NAME}" from "${OVNKUBE_CONTROLLER_POD}"..."
+      oc cp openshift-ovn-kubernetes/"${OVNKUBE_CONTROLLER_POD}":/etc/ovn/ovn"${DB}"b_db.db -c "${CONTAINER}" \
+        "${NETWORK_LOG_PATH}/${OVNK_DATABASE_STORE_PATH}/${OVNKUBE_CONTROLLER_POD}_${DB}bdb" 2>&1 & PIDSDB+=($!)
```

Comment:
NIT: does PIDSDB have a limit? At some point, we will need to worry about how many nodes could be going at once, but that can be done at a different PR.